plugins/linear-genome-view/src/LinearGenomeView/model.ts

```diff
@@ -345,6 +345,25 @@ export function stateModelFactory(pluginManager: PluginManager) {
           ...new Set(self.displayedRegions.map(region => region.assemblyName)),
         ]
       },
+      /**
+       * #getter
+       */
+      get assemblyDisplayNames() {
+        const session = getSession(self)
+        const { assemblyManager } = session
```

Comment:
possibly can leverage the existing "get assemblyNames" getter above:

```
const session = getSession(self)
        const { assemblyManager } = session
        return this.assemblyNames.map(assemblyName => {
          const assembly = assemblyManager.get(assemblyName)
          return assembly?.displayName ?? assemblyName
        })
```

I'm also

Reply:
note: in the fix it does assembly.displayName??assembly.name but on assembly.displayName getter already does displayName??name